src/System.Slices/System/ReadOnlySpan.cs

```diff
@@ -336,18 +333,5 @@ public override bool Equals(object obj)
             }
             return false;
         }
-
-        /// <summary>
-        /// Returns item from given index without the boundaries check
-        /// use only in places where moving outside the boundaries is impossible
-        /// gain: performance: no boundaries check (single operation) 
-        /// </summary>
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal T GetItemWithoutBoundariesCheck(int index)
-        {
-            return PtrUtils.Get<T>(Object, Offset, (UIntPtr)index);
-        }
```

Comment:
I removed  `GetItemWithoutBoundariesCheck` -  it is not inlined for reference types.